src/Gutenberg/BlockParserInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace WpOop\WordPress\Gutenberg;
+
+/**
+ * The interface for parsing Gutenberg blocks.
+ */
+interface BlockParserInterface
+{
+    /**
+     * @param string $postContent Content of a WP post (e.g. WP_Post post_content).
+     * @return BlockInterface[]
+     */
+    public function parseBlock(string $postContent): array;
```

Comment:
If it's suppposed to return an arbitrary number of blocks, can we call it `parseBlocks()` (plural)?

Reply:
ah, yeah.